array/DNAm/preprocessing/jobSubmission/1_runDNAmQC.sh

```diff
@@ -10,33 +10,38 @@
 #SBATCH --output=DNAm/logFiles/%u/QCDNAdata.log # output file
 #SBATCH --job-name=QCDNAdata
 
+#------------------------------------------------------
+
+# 1. the first input is your project folder.
+# 2. the second input is the full path to config file.
+# 3. the third input is a the path to the config.r file.
+
+#-----------------------------------------------------
+
+
 ## print start date and time
 echo Job started on:
 date -u
 JOBNAME="QCDNAdata"
-    
+
+# Forcibly moves the user to the scripts directory
+cd $(dirname $0)/../../../..
```

Comment:
Although this looks a bit ugly, I can't think of a better way of doing this. It will have to do. I guess another way of doing this would be to ask the user to put the location of the repository into the config file?